src/CoilMaps.jl

```diff
@@ -33,7 +29,18 @@ function calcCoilMaps(data::AbstractArray{Complex{T},3}, trj::AbstractVector{<:A
         @views mul!(xbp[img_idx,icoil], adjoint(p), vec(dataU))
     end
     verbose && println("BP for coils maps: $t s")
+    return xbp
+end
+
 
+function calcCoilMaps(data::AbstractArray{Complex{T},3}, trj::AbstractVector{<:AbstractMatrix{T}}, U::AbstractMatrix{Complex{T}}, img_shape::NTuple{N,Int}; density_compensation::Union{Symbol, <:AbstractVector{<:AbstractVector{T}}}=:radial_3D, kernel_size = ntuple(_->6, N), calib_size =  ntuple(_->24, N), eigThresh_1=0.01, eigThresh_2=0.9, nmaps=1, verbose = false) where {N,T}
+    Ncoils = size(data,3)
+    Ndims = length(img_shape)
+    imdims = ntuple(i->i, Ndims)
+
+    xbp = calcFilteredBackProjection(data, trj, U, img_shape, Ncoils, density_compensation=density_compensation, verbose = verbose)
```

Comment:
Little cosmetic change: I recently learned that the style guide suggests using a semicolon before the first keyword argument. Additionally, you can actually shorten the keyword argument calls if the argument and the argument's name are identical:
```
xbp = calcFilteredBackProjection(data, trj, U, img_shape, Ncoils; density_compensation, verbose)
```
In this case, the semicolon is required. Otherwise, it looks good!